modules/lang-painless/src/main/java/org/elasticsearch/painless/api/Augmentation.java

```diff
@@ -181,8 +181,13 @@ public static String namedGroup(Matcher receiver, String name) {
      */
     public static <T> String join(Iterable<T> receiver, String separator) {
         StringBuilder sb = new StringBuilder();
+        boolean firstToken = false;
```

Comment:
@nagads, could you hit this with the formatter? It should put the `else` on the same line and remove the empty line. At some point we're going to make this a requirement for everything just so you don't have to think too much about formatting when hacking on ES. But we haven't yet.

Reply:
@nik9000 thanks for kind feedback, please find the format fix in latest commit.